CLAUDE.md

```
@@ -303,3 +303,245 @@ CI uses JDK 17 for standard builds and JDK 21 for release builds. Gradle caches
+
+#### Phase 0: Bootstrap (already done — Ruflo initialized)
+
+```bash
+npx ruflo@latest memory search -q "otaku-reader audit"   # retrieve prior findings
+npx ruflo@latest memory stats                             # check DB health
+```
+
+#### Phase 1: Architecture & Structural Mapping
+
+**Ruflo (with MCP registered):**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)